test-site/.env.plugins

```diff
@@ -1,6 +1,6 @@
 WORDPRESS_BASIC_AUTH=YWRtaW46c2VjcmV0
 WPGRAPHQL_VERSION=v1.0.1
-WPGATSBY_VERSION=feature/wpgatsby-stability
+WPGATSBY_VERSION=master
```

Comment:
oops! good catch

Reply:
Just setting it temporarily :) I'm going to release a new WPGatsby version now and then update this PR once it's published